lib/model.js

```diff
@@ -1375,6 +1375,12 @@ Model.createCollection = function createCollection(options, callback) {
   this.schema.options.timeseries;
   if (timeseries != null) {
     options = Object.assign({ timeseries }, options);
+    if (this.schema.options.expireAfterSeconds) {
```

Comment:
Let's add a check for `if (options.expireAfterSeconds != null)`. We don't want the schema option to override user-provided options if someone calls `createCollection({ expireAfterSeconds })`.

So I think this should look like:

```javascript
if (options.expireAfterSeconds != null) {
  // ...
} else if (options.expires != null) {
  // ..
} else if (this.schema.options.expireAfterSeconds != null) {
  // ...
} else if (this.schema.options.expires != null) {
  // ...
}
```

Reply:
Note to myself: I guess typings of parameter options of createCollection has also be adapted.

Reply:
done